### ISSUE TITLE
fail to install the library following the instruction..

### ISSUE THREAD

**Issue by Clarkszw** [qcscine/molassembler#11]
Hi Expert,
Need some help to install the library.. I have tried to follow the instruction. But no luck to success..

## CMake
Could you elaborate _"When building with CMake, Boost and Eigen must be installed and available via CMake's find_package (e.g. via CMAKE_PREFIX_PATH). "_
I have downloaded Boost and Eigen, but I don't find out how to link it to CMake.. (via modifying `CMakeLists.txt`?)
Here is the output when I try to run `cmake -DCMAKE_BUILD_TYPE=Rlease ..` in the `build-release` directory:
```
CMake Warning at dev/cmake/ComponentSetup.cmake:61 (message):
  You are compiling Scine components with an architecture-specific ISA:
  -march=native.  Linking together libraries with mismatched architecture
  build flags can cause problems, in particular with Eigen.  Watch out!
Call Stack (most recent call first):
  build-release/scine-utils-os-src/CMakeLists.txt:16 (scine_setup_component)


CMake Warning at /usr/share/cmake/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  build-release/scine-core-src/src/Core/CMakeLists.txt:3 (find_package)


CMake Warning at /usr/share/cmake/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  build-release/scine-core-src/src/Core/CMakeLists.txt:3 (find_package)


CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Boost (missing: system filesystem) (found version "1.82.0")
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindBoost.cmake:2344 (find_package_handle_standard_args)
  build-release/scine-core-src/src/Core/CMakeLists.txt:3 (find_package)


-- Configuring incomplete, errors occurred!
See also "/anfhome/a-zhiwenshi/azurequantumnotebookdemos/aiida_pyscf/ie/molassembler/build-release/CMakeFiles/CMakeOutput.log".
```

## conan
Here is the output when I run `conan install -r scine --build=missing scine_molassembler/2.0.0@`
```
ERROR: Conanfile not found at /anfhome/a-zhiwenshi/azurequantumnotebookdemos/aiida_pyscf/ie/scine_molassembler/2.0.0@
```
seems missing Conanfile to configure the process?..(how to create then?)

## PyPI
run `python3 -m pip install scine-molassemler` gives me:
```
ERROR: Could not find a version that satisfies the requirement scine-molassemler (from versions: none)
ERROR: No matching distribution found for scine-molassemler
```
I have checked https://pypi.org/project/scine-molassembler/#files for the release. It shows another package name with `-` instead of `_`. But it gives me the same response when I run `pip install scine-molassembler`:
```
ERROR: Could not find a version that satisfies the requirement scine-molassembler (from versions: none)
ERROR: No matching distribution found for scine-molassembler
```
----
I am really interested in the library, but get frustrated in installation.. HELP.. :)

**Comment by weymutht** [comment]
Thanks for your interest in Molassembler! Getting the installation from PyPI to work is probably easiest, so I suggest to focus on that first. May I kindly ask what Python version you're using? I'm suspecting that you're using a version for which we currently don't supply a package, but that would be easy to fix for us.

**Comment by Clarkszw** [comment]
Hi @weymutht, thanks for your quick response.
I am using Python 3.11.4. I can see the prebuilt wheels are for cp3.6-cp.3.9, I can also try on python 3.9 first:)

**Comment by Clarkszw** [comment]
After installing another dependency `scine-utilities-python` from `conda-forge`, I have successfully installed `scine-molassembler` via pip with Python 3.9.16

**Comment by weymutht** [comment]
Great to hear! Please note that the packages on conda forge are provided by a third party. If I'm correct, the most recent version of the utilities package on conda forge is 6.0.0, which is a bit behind the most recent official release (version 8.0.0). I recommend to rely on PyPI if possible.

(I think there is not yet a package for the utilities for Python 3.9 available. I could easily provide you if you decide to stick to Python 3.9.)

**Comment by Clarkszw** [comment]
yes, `scine-utilities-python` is 6.0.0
Could you provide me the package for python 3.9. Or if it is easier, you can guide me how to install it in another python version? (did I simply downgrade to, fx 3.8, to install the latest utilities package?

**Comment by weymutht** [comment]
Packages for Python 3.9 and 3.10 are now available for version 8.0.0 on PyPI. (Downgrading to Python 3.8 would also have been a possibility.)
